Client/Ecosia/Migration/EcosiaFavourites.swift

```diff
@@ -59,3 +61,10 @@ final class EcosiaFavourites {
     }
 
 }
+
+private extension Core.Page {
+    var urlString: String? {
```

Comment:
what about adding a unit-test for this accessor?

Reply:
Sure